### ISSUE TITLE
Inconsistent warnings on rspec "should" idiom

### ISSUE THREAD

**Issue by dpassage** [rubocop/rubocop#307]
I have the following code in my test specs:

```
  describe "#drivers_by_qualifying_points" do
    it "returns an array of driver entries by qualifying points" do
      season = seasons(:season_2002)
      results = season.drivers_by_qualifying_points
      results.length.should == 12                                #this is line 98
      results[0].entrant.should == drivers(:schumacher)
      results[0].points.should == 67
    end
  end
```

rubocop 0.8.3 reports the following warnings:

```
W: 98: Possibly useless use of == in void context
W: 99: Possibly useless use of == in void context
```

The biggest bug is the inconsistency; rubocop should be reporting the same thing for all 3 lines. Ideally that would be not printing a warning.
